h2i requires terminal.MakeRaw and terminal.Restore which are not currently
available on Solaris.

Not suitable for upstream.
diff -ru net-b4e17d61b15679caf2335da776c614169a1b4643/http2/h2i/h2i.go net-b4e17d61b15679caf2335da776c614169a1b4643.new/http2/h2i/h2i.go
--- net-b4e17d61b15679caf2335da776c614169a1b4643/http2/h2i/h2i.go	2015-11-02 02:32:43.000000000 -0800
+++ net-b4e17d61b15679caf2335da776c614169a1b4643.new/http2/h2i/h2i.go	2015-11-03 14:50:26.289697762 -0800
@@ -1,6 +1,7 @@
 // Copyright 2015 The Go Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
+// +build !solaris
 
 /*
 The h2i command is an interactive HTTP/2 console.
